helm/vertical-pod-autoscaler-app/templates/admission-controller-certgen.yaml

```diff
@@ -72,7 +72,7 @@ spec:
             set -o errexit
             set -o nounset
             set -o pipefail
-            apk --update add openssl
+            apk --update add openssl || true
```

Comment:
Why isn't the [original fix](https://github.com/giantswarm/retagger/pull/758) enough? I saw that the change was rolled back and replaced with this [one](https://github.com/giantswarm/vertical-pod-autoscaler-app/pull/135/files#diff-1b09f59b5f76f8cdfaeb4a5a5481c722c847c7884b9e516578b77819befbf96cR75). I don't think we should keep bad practices just because upstream does the same.

Also, I don't know if the fix works because what happens if openssl is not already present in the image? How would the certificates be generated? If openssl is already present, than there is no need to run apk.

Reply:
I think this fix ensure that openssl is always uptodate to avoid CVEs but the upstream we currently use is deprecated :(

Reply:
We made a point with @TheoBrigitte and we propose:

1. removing `apk update` from the manifest as `openssl` already exists in v11-alpine image
2. we will use chart dependency in `vertical-autoscaler-app` from this upstream repo: https://github.com/cowboysysop/charts/tree/master/charts/vertical-pod-autoscaler and no `openssl` update is done